README.md

```diff
@@ -50,28 +50,36 @@ releases of the toolkit.
-      file is provided. To establish the conda environment locally, [Conda](https://conda.io/docs/index.html) must first
+* To run GATK:
+    * Java 8
+    * Python 2.6 or greater (required to run the `gatk` frontend script)
+    * Python 3.6.2, along with a set of additional Python packages, is required to run some tools and workflows.
+      GATK uses the [Conda](https://conda.io/docs/index.html) package manager to establish and manage the
+      environment and dependencies required by these tools. The GATK Docker image comes with this environment
+      pre-configured. In order to establish an environment suitable to run these tools outside of the Docker image, the
+      conda [gatkcondaenv.yml](https://github.com/broadinstitute/gatk/blob/master/scripts/gatkcondaenv.yml) file is
+      provided. To establish the conda environment locally, [Conda](https://conda.io/docs/index.html) must first
       be installed. Then, create the gatk environment by running the command ```conda env create -n gatk -f gatkcondaenv.yml```
       (developers should run ```./gradlew createPythonPackageArchive```, followed by
       ```conda env create -n gatk -f scripts/gatkcondaenv.yml``` from within the root of the repository clone).
       To activate the environment once it has been created, run the command ```source activate gatk```. See the
       [Conda](https://conda.io/docs/user-guide/tasks/manage-environments.html) documentation for
       additional information about using and managing Conda environments.
     * R 3.2.5 (needed for producing plots in certain tools, and for running the test suite)
-    * [git-lfs](https://git-lfs.github.com/) 1.1.0 or greater (needed to download large files for the complete test suite).
-      Run `git lfs install` after downloading, followed by `git lfs pull` from the root of your git clone to download the large files. The download is several hundred megabytes.
-* Alternatively, pre-packaged images with all needed dependencies installed can be found on [our dockerhub repository](https://hub.docker.com/r/broadinstitute/gatk/). This requires a recent version of the docker client, which can be found on the [docker website](https://www.docker.com/get-docker).
+* To build GATK:
+    * A Java 8 JDK
+    * Git 2.5 or greater
+    * [git-lfs](https://git-lfs.github.com/) 1.1.0 or greater. Required to download the large files used to build GATK and
+      run the test suite. Run `git lfs install` after downloading, followed by `git lfs pull` from the root of your git clone
+      to download all of the large files. The full download is approximately 2 gigabytes. Alternatively, to pull the minimal
+      set of large `lfs` resource files required to only build ()without running tests), the test resources can be excluded
+      from the download by running `git lfs pull --include src/main/resources/large/` instead of `git lfs pull`. This greatly
```

Comment:
Can we modify `build.gradle` to run this command automatically at build time, if lfs stubs are detected? Let's do that (if possible) -- then this set of instructions can be simplified to just "install git lfs"

Reply:
Done. I changed the build to check for stubs, and to only run `git lfs pull --include src/main/resources/large/` if it finds any, then validate again. Also, we still have to manually download the files into the staging area, since the Docker doesn't have git-lfs installed.

Reply:
Done see above note.